src/sidebar/store/modules/selection.js

```diff
@@ -395,26 +419,35 @@ function setSortKey(key) {
   };
 }
 
-/** Is the annotation referenced by `$tag` currently focused? */
-function isAnnotationFocused(state, $tag) {
-  return state.selection.focusedAnnotations.some(
-    focusedAnn => $tag && focusedAnn === $tag
-  );
+/* Selectors */
+
+function focusedAnnotations(state) {
+  return truthyKeys(state.selection.focused);
```

Comment:
Depending on how often these selectors are invoked per render, you might want to use `createSelector` to memoize the result. eg.

```
const focusedAnnotations = createSelector(
  state => state.selection.focused,
  focused => truthyKeys(focused))
);
```

Reply:
And done...